### PR TITLE
Add Jenkinsfile trigger syntax example

### PR DESCRIPTION
**Pull request by johnbieren**
Especially with the syntax changing with version 1.1.1, I think it is good to have an example to work from.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -32,3 +32,27 @@ To run tests:
 ## CI
 
 This plugin is currently built on [ci.jenkins.io](https://ci.jenkins.io/job/Plugins/job/jms-messaging-plugin/).
+
+## Jenkinsfile Build Trigger Example
+
+This example shows triggering your pipeline from Fedmsg with the org.fedoraproject.prod.buildsys.build.state.change topic
+
+````
+pipelineTriggers(
```
Comment (scoheb) [inline README.md]: This is in the context of the properties step in a Jenkinsfile?

Comment (johnbieren) [reply README.md]: That is correct. I can put that if you want; pipelineTriggers going in properties was found by me on google, but syntax for jms triggering was not. But yeah, do you think it makes it more complete? If so, I'll add it.

Comment (scoheb) [reply README.md]: I would add it to be more complete.

By the way, have you tried the ```Pipeline Syntax``` page?

Comment (johnbieren) [reply README.md]: Never thought to look under properties for the trigger

**Pull request closed without merging**